spec/factories/settings.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: settings
+#
+#  id    :integer          not null, primary key
+#  key   :string
+#  value :string
+#
+
+FactoryBot.define do
+  factory :setting do
+    key   { 'key' }
+    value { 'value' }
```

Comment:
I would use Faker here, what do you think?

Reply:
I won't because there is no specific one to match, also which would be the benefit of using it?

Reply:
Anyone now, but will prevent to change it in a future if we need to add extra settings and validations between them.

Reply:
You should never use the default factory (only for simple model restrictions). If you are using this on a feature test is cause you want to test a specific key, so I don't find any reason to make this variable with Faker